src/commands/url-to-payid.ts

```diff
@@ -0,0 +1,32 @@
+import * as Vorpal from 'vorpal'
+
+import * as utils from '../index'
+
+import Command from './Command'
+
+/**
+ * Converts a url (eg https://xpring.money/test) to a PayID (eg test$xpring.money).
+ */
+export default class UrlToPayidCommand extends Command {
+  /**
+   * @override
+   */
+  protected command(): string {
+    return 'url to-payid <url>'
```

Comment:
Since this is the only `url` command, does it make sense to make this: `payid from-url`?

Reply:
Makes sense.

Reply:
Updated to `payid from-url`